File: scripts/vml-out-probe/langs/out.kt
// out.kt —— VML 跨语言「输出」判据（期望恰好三行，见 run-langs.sh）
// 写法照 corpus/kt/skel.* —— 共享库同时提供 println_str / println_int。
fun main() {
    println_str("OUT-STR=abc")
    print_str("OUT-INT=")
    println_int(42)
    println_str("OUT-PUN=hello, world")
}

File: scripts/vml-out-probe/langs/out.go
// out.go —— VML 跨语言「输出」判据（期望恰好三行，见 run-langs.sh）
package main

func main() {
	print_str("OUT-STR=abc")
	println_int(0)
	println_int(42)
	print_str("OUT-PUN=hello, world")
	println_int(0)
}
